pkg/server/ca/ca.go

```diff
@@ -97,7 +99,14 @@ func (ca *serverCA) SignX509SVID(ctx context.Context, csrDER []byte, ttl time.Du
 		return nil, err
 	}
 
-	ca.c.Log.Debugf("Signed x509 SVID %q (expires %s)", cert.URIs[0].String(), cert.NotAfter.Format(time.RFC3339))
+	spiffeID := cert.URIs[0].String()
+	ca.c.Log.Debugf("Signed x509 SVID %q (expires %s)", spiffeID, cert.NotAfter.Format(time.RFC3339))
+	ca.c.Metrics.IncrCounterWithLabels([]string{"server_ca", "sign", "x509_svid"}, 1, []telemetry.Label{
```

Comment:
are these counters emitted with `spire_server` prefix, or `spire.server.` or something along those lines? We should probably do that if we're not, in which case this can probably go from `server_ca` to just `ca`?

Reply:
The service name (i.e "spire_server" or "spire_agent") is prefixed to keys unless you set EnableServiceLabel to `true`. The default value (which we don't change) is false.

Reply:
Awesome... thoughts on `server_ca` -> `ca`?